samples/react/package.json

```diff
@@ -113,7 +113,7 @@
     "bootstrap:connected": "node scripts/bootstrap.js",
     "bootstrap:disconnected": "node scripts/bootstrap.js --disconnected",
     "graphql:update": "node -r babel-register ./scripts/update-graphql-fragment-data.js",
-    "test": "react-scripts test --env=jsdom",
+    "test": "cross-env-shell CI=true \"react-scripts test --env=jsdom\"",
```

Comment:
CI=true - is this intentional?

Reply:
Without this change tests are hanging on CI for react app which we are not changing.